php/libraries/EntityType.class.inc

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+/**
+ * This defines the entity type for a Candidate. It is a Value Object used
+ * to assist with validation.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * This defines the entity type for a Candidate. It is a Value Object used
+ * to assist with validation.
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class EntityType
+{
+    /* @var string */
+    $value = '';
```

Comment:
This is missing an access modifier.. I didn't think it was even valid without either an access modifier or the `var` keyword.

Reply:
And there is non need for a default value.